src/pages/Summary/index.tsx

```diff
@@ -157,6 +174,60 @@ const _SummaryPage = ({
           />
         </span>
       </AcknowledgementSection>
+      <AcknowledgementSection title="Protect yourself against double deposits">
+        {beaconChainApiStatus === BeaconChainStatus.DOWN && (
+          <Alert variant="warning" className="mb20">
+            <div className="flex">
+              <AlertIcon />
+              <Text
+                weight={500}
+                color="yellowDarkest"
+                className="my10"
+                style={{ wordBreak: 'break-word' }}
+              >
+                Proceed with caution. Our on-chain data source is down and we
+                are unable to flag any double deposits.
+              </Text>
+            </div>
+          </Alert>
+        )}
+
+        <Text>
+          Again, <i>you are responsible for this transaction!</i> Duplicate
+          deposits with the same keyfile public key will not be rejected, but
+          your funds will be lost.
+        </Text>
+        <ul>
+          <li>
+            <Text className="mt10">
+              Do not submit any transaction with a deposit_data file that you
+              did not create yourself, or that you do not own the mnemonic
+              phrase for.
+            </Text>
+          </li>
+          <li>
+            <Text className="mt10">
+              If you've recently submitted a transaction with this deposit_data
+              file, and are now resubmitting for some reason, we recommend
+              waiting at least 30 minutes for our on-chain data source to flag
+              any duplicates.
+            </Text>
+          </li>
+        </ul>
+
+        <span className="mt20">
+          <CheckBox
+            onChange={e => setDuplicatesAcknowledged(e.target.checked)}
+            checked={duplicatesAcknowledged}
+            label={
+              <Text>
+                I understand that sending a duplicate transaction will result in
+                a loss of funds.
```

Comment:
I'm not sure if it's 100% correct to say it's "a loss of funds" because technically it's a top-up. (and that's another functionality we want to provide on the launchpad with another flow)

@CarlBeek any thoughts?

Reply:
@CarlBeek, how about:

"I understand that sending a deposit more than once will result in my validator's balance exceeding 32 ETH, and that there is no advantage to this since a validator's maximum stake is limited to 32 ETH."